src/auth/choicewindow.cpp

```diff
@@ -0,0 +1,89 @@
+#include "choicewindow.h"
+#include "downloaderWrapper.h"
+
+#include <QLabel>
+#include <QListWidget>
+#include <QVBoxLayout>
+#include <QDialogButtonBox>
+#include <QAbstractButton>
```

Comment:
`#include <QAbstractButton>` 
you don't need it, when you included 
`#include <QPushButton>`

Reply:
removed